app/domain/urls.ts

```diff
@@ -413,6 +402,17 @@ const localeAwareUrls: Urls = {
     apiDocs: '/api-docs',
     statistics: '/statistics',
     limit: '/limit'
+  },
```

Comment:
Could you move this back up so that we keep it sorted? The same goes with the mapping in `flag.ts`

Reply:
Should I request a new PR? How do I edit it in #547? (Same as for the config)

Reply:
I don't have write access

Reply:
Already fixed it: https://github.com/BustByte/coronastatus/pull/547/commits/65ed0becbd5501c4ab260523a3f494035520f123

Reply:
Copy thanks! @fossecode